prow/e2e-cluster_wide-auth.sh

```diff
@@ -49,4 +49,4 @@ get_resource "${RESOURCE_TYPE}" "${OWNER}" "${INFO_PATH}" "${FILE_LOG}"
 setup_cluster
 
 echo 'Running cluster-wide e2e rbac, auth Tests'
-${ROOT}/prow/e2e-suite-rbac-auth.sh --cluster_wide "$@"
+${ROOT}/prow/e2e-suite.sh --auth_enable --cluster_wide "$@"
```

Comment:
Should we change the title since this seems to enable VM test?

Reply:
no if you look at e2e-suite-rbac-auth.sh, it s calling ./prow/e2e-suite.sh --test_vm --auth_enable "$@"
I am just calling ./prow/e2e-suite.sh  directly and dropping the --test_vm